_data/meltano/extractors/tap-spreadsheets-anywhere/ets.yml

```
@@ -53,8 +53,8 @@ settings:
     - prefer_number_vs_integer: (optional) If the discovery mode sampling process sees only integer values for a field, should number be used anyway so that floats are not considered errors? The default is false but true can help in situations where floats only appear rarely in sources and may not be detected through discovery sampling.
     - selected: (optional) Should this table be synced. Defaults to true. Setting to false will skip this table on a sync run.
     - worksheet_name: (optional) the worksheet name to pull from in the targeted xls file(s). Only required when format is excel
-    - delimiter: (optional) the delimiter to use when format is 'csv'. Defaults to a comma ',' but you can set delimiter to 'detect' to leverage the csv "Sniffer" for auto-detecting delimiter.
-    - quotechar: (optional) the character used to surround values that may contain delimiters - defaults to a double quote '"'
+    - delimiter: (optional) the delimiter to use when format is 'csv' e.g. ','. You can leave delimiter blank or set it to 'detect' to leverage the csv "Sniffer" for auto-detecting delimiter.
```

I've found that this is more accesible

```suggestion
    - delimiter: (optional) the delimiter to use when format is 'csv', for example ','. You can leave delimiter blank or set it to 'detect' to leverage the csv "Sniffer" for auto-detecting delimiter.
```